### ISSUE TITLE
Is this project deprecated?

### ISSUE THREAD

**Issue by beddueimpossibile** [FieldCommGroup/WindowsHartIpClient#26]
Hi everybody,

I found this project and started using it to begin testing my custom HART-IP server, but I hit the roof already because this client is not full featured, is still supporting only V1 and the development stopped somewhere in the late 2019.

Meanwhile s new official repo appeared [here](https://github.com/FieldCommGroup/Portable-HART-IP-Client) and I am wondering if i should be looking at that due to this one being abandoned, or if they are two different projects with different aims and resuming this one kinda makes sense.

Thanks

**Comment by tjohnston-fcg** [comment]
Yes you should.  The v2 client is not available as source.  It ships with the latest HART Test System and also the paid HART-IP Developer Kit.

**Comment by beddueimpossibile** [comment]
Hello @hswhilden 

Can you please share your contact info? Thanks
